_maps/DKsJungleAdventureC/DKs-Jungle-AdventureC.yaml

```diff
@@ -23,7 +23,7 @@ tourMode:
 background: bg109
 mapIcon: dksjungle
 music:
-  download: https://drive.google.com/u/0/uc?id=1dyaF7ZulJ-BAp1lountI4dewRE3goRN3&export=download
+  download: https://drive.google.com/u/0/uc?id=1dyaF7ZulJ-BAp1lountI4dewRE3goRN3&export=download&confirm=1
```

Comment:
Ah yes, I realised the music download link for DK's Jungle Adventure was redundant since it's grabbing music from the Background download. However, DKs-Jungle-AdevntureC is the Console version without the background, so it needs a music download link. Should be s3, but we can fix that later when Nikki gives me s3 access.

Reply:
QuietStreetC is the same deal.